src/main/java/bio/terra/workspace/service/status/WorkspaceManagerStatusService.java

```diff
@@ -41,6 +43,7 @@ public WorkspaceManagerStatusService(
           new StatusSubsystem(checkDataRepoInstanceFn, /*isCritical=*/ false));
     }
 
+    registerSubsystem("Buffer", new StatusSubsystem(bufferService::status, /*isCritical=*/ true));
```

Comment:
aside: It would be better to have a `criticality` or `Severity` enum instead of this boolean, for future expansion and readability.